tutorials/basic_1_square/README.md

```diff
@@ -0,0 +1,137 @@
+  static const int square_speed = 400;
+
+  // To represent our square we are using the Rect class from dart:ui
+  // which is a handle class to represent this type of data. We will be
+  // seeing other types of data classes in the future, but for this
+  // example, Rect will do fine for us.
+  late Rect squarePos;
+
+  // To reprent our direction, we will be using an int value, where 1 means
+  // going to the right, and -1 going to the left, this may seems like a too much
+  // simple way of representing a direction, and indeed it is, but this will
+  // will work fine for our small example and will make more sense when we implement
+  // the update method
+  int squareDirection = 1;
+
+  // The onLoad method is where all of the game initialization is supposed to go
+  // for this example, you may think that this square could just be initialized on the field
+  // declaration, and you are right, but for ludic reasons and to present the life cycle method
+  // for this example we will be initializing this field here.
+  @override
+  Future<void> onLoad() async {
+    squarePos = Rect.fromLTWH(0, 0, 100, 100);
+  }
+
+  // Update and render ommited
+}
+```
+
+Right, now we have all the data and variables we need to start implementing our game, for the next step, lets just draw our little square on the screen.
+
+```dart
+  @override
+  void render(Canvas canvas) {
+    // Canvas is a class from dart:ui and is reponsible for all the renderizations on Flame
+    //
+    // BasicPalette is a help class from Flame, which provides default, pre builded instances
+    // of Paints that can be used by your game
+    canvas.drawRect(squarePos, BasicPalette.white.paint);
+  }
+}
+```
+
+You may now be seeing a static white square being rendered on the top left corner of your screen, which is not that cool, so, to finish our example, lets add some movement to the game and implement our update method:
+
+```dart
+  @override
+  void update(double dt) {
+    // Here we move our square by applying our iteration delta time (dt) to our
+    // speed variable and direction. Note that the Rect class class is immutable
+    // and the translate method returns a new Rect instance for us, so we just
+    // re assign it to our square variable
+    squarePos = squarePos.translate(square_speed * squareDirection * dt, 0);
+
+    // This simple verifies if the square is going right, and has reached the end of the screen
+    // if so, we just invert our direction
+    if (squareDirection == 1 && squarePos.right > size.x) {
+      squareDirection = -1;
+    // This does the same, but now checking the left direction
+    } else if (squareDirection == -1 && squarePos.left < 0) {
+      squareDirection = 1;
+    }
+  }
+```
+
+And that is it for this basic tutorial, here we have covered the basics of Flame, its basic classes and some basic rendering. From that we can start to build more complex things and more "exciting" games.
```

Comment:
basic tutorial, in which we have

basic basic basic

I dont think we need to quote exciting, sounds a bit patronizing

Reply:
Fixed first and last.

Any suggestions for basic basic basic?

Reply:
up to you. its not a really big issue anyway